### ISSUE TITLE
Details of training P

### ISSUE THREAD

**Issue by eugenelyj** [Doubiiu/DynamiCrafter#110]
Dear authors, Could you share some details of training P, including 1) training iteration, 2) learning rate, 3) weight initialization ?

**Comment by eugenelyj** [comment]
![image](https://github.com/user-attachments/assets/708e16b3-3154-4429-bdb5-110ed0baca41)
Found in the paper :)
